lib/ansible/modules/cloud/azure/azure_rm_cdnprofile.py

```diff
@@ -30,11 +30,11 @@
         required: true
     location:
         description:
-            - Valid azure location. Defaults to location of the resource group.
+            - Valid Azure location. Defaults to location of the resource group.
     sku:
         description:
-            - The pricing tier, defines a CDN provider, feature list and rate of the CDN profile.
-            - Detailed pricing can be find at U(https://azure.microsoft.com/en-us/pricing/details/cdn/)
+            - The pricing tier (defines a CDN provider, feature list and rate) of the CDN profile.
```

Comment:
no need to change

Reply:
Thanks yuwzho, This is a change based on the [SDK description](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-cdn/azure/mgmt/cdn/models/sku_py3.py). I will change it according to your requirement.